arangodb-net-standard/IndexApi/Models/InvertedIndexResponse.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace ArangoDBNetStandard.IndexApi.Models
+{
+    /// <summary>
+    /// Response from <see cref="IIndexApiClient.PostInvertedIndexAsync(PostIndexQuery, PostInvertedIndexBody, System.Threading.CancellationToken)"/>
+    /// </summary>
+    public class InvertedIndexResponse : PostInvertedIndexBody
```

Comment:
It's strange to see a response class inherit from a request body. Maybe create an interface with the common properties instead?

Reply:
I created the interface for the common properties.